src/Terrain/Environment.java

```diff
@@ -13,7 +13,7 @@
   
   public enum Weathers
   {
-    CLEAR, RAIN, SNOW, SANDSTORM
+    CLEAR, SNOW, RAIN, SANDSTORM
```

Comment:
OK, I gotta ask; why?

Reply:
I was originally going to implement a priority system: clear is the "weakest", snow is superseded by other weather types (since it's almost strictly stronger than rain), and sandstorm is defined by the removal of moisture, which kinda deflates the other weather types.
On more thought, it didn't make a lot of sense, but I'd already made the commit.
And it doesn't affect anything, so I didn't revert.